assets/scripts/main.js

```diff
@@ -36,6 +37,15 @@ if (window.location.hostname === 'streetmix.net' || window.location.hostname ===
   }).install()
 }
 
+function unsupportedBrowser () {
+  const browser = detect()
+  const { unsupportedBrowser } = system
+  if (browser.name === unsupportedBrowser.name && browser.version === unsupportedBrowser.version) {
```

Comment:
Just a question here, is this checking just if the browser version is equal to IE8? What about older versions of IE, shouldn't the same message be displayed?

Reply:
Yes -- you're correct about this. This particular function is _only_ testing if the browser is IE8. We need another approach.

Reply:
It's a little more complicated than that. The version numbers reported back can be in the format like "8.0.0". A simple mathematical comparison won't work in this case.

Reply:
Yeah true, we could keep an array of unsupported versions and do an ```indexOf()``` or if this "8.0.0" format is constant, we could split it into an array and perform logic on each value although that will mean more code.